remote: add description for verify option (#937)
iterative/dvc.org: public/static/docs/command-reference/remote/modify.md

- `-v`, `--verbose` - displays detailed tracing information.

> ## Avialable settings for all remotes

Reviewer: ```suggestion
## Available settings for all remotes
```
p.s. consider using a spell checker in your IDE.

Reviewer: I'll fix this.

> The following options are avialable for all remotes:

Reviewer: ```suggestion
The following options are available for all remote types:
```

- `verify` - upon downloading cache files (`dvc pull`, `dvc fetch`) DVC will

>   recalculate downloaded file checksum to make sure that it is not corrupted or

Reviewer: ```suggestion
  recalculate downloaded file checksums to make sure that these are not corrupted or
```

>   modified during download. It may slow down mentioned commands.

Reviewer: ```suggestion
  modified during download. It may slow down the aforementioned commands.
```
